src/www/ui/api/documentation/openapi.yaml

```
@@ -3704,6 +3729,25 @@ components:
           type: boolean
           description: Add the copyright information as text findings
           default: false
+    AdminLicenseAcknowledgement:
+      type: object
+      properties:
+        la_pk:
```

Would suggest calling it ID.
```suggestion
        id:
```

---

Done